applications/nrf5340_audio/src/bluetooth/bt_management/dfu/bt_mgmt_dfu.c

```diff
@@ -82,15 +84,17 @@ static void dfu_set_bt_name(void)
 
 #endif
 	strlcat(name, "_DFU", CONFIG_BT_DEVICE_NAME_MAX);
-	bt_set_name(name);
+
+	sd_peer[0].type = BT_DATA_NAME_COMPLETE;
+	sd_peer[0].data_len = (sizeof(name) - 1);
```

Comment:
This will always return `CONFIG_BT_DEVICE_NAME_MAX - 1`. Not sure you want strlen? (Not with -1 as '\0' is not counted by that function.

Reply:
Used strlen now